src/app/containers/LoginContainer.js

```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import { c } from 'ttag';
-import { LoginForm, Href, SimpleDropdown, DropdownMenu, SignInLayout } from 'react-components';
+import { LoginForm, Href, SimpleDropdown, DropdownMenu, SignInLayout, useUser } from 'react-components';
 
 const LoginContainer = ({ stopRedirect, history, onLogin }) => {
+    const [user] = useUser();
```

Comment:
You cannot use user in the public app, but you can destructure the raw user object from ...args I think

Reply:
@mmso @vincaslt Can we use the `User` returned directly by the API call then during the login?

Reply:
Yeah you can destructure User from ...args

Reply:
```js
import { isSubUser } from 'proton-shared/lib/user/helpers';
const { User = {} } = args;

if (isSubUser(User)) {
 // ...
}
```